roles/engine_setup/tasks/restore_engine_from_file.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Run engine cleanup command
+  command: "engine-cleanup"
```

Comment:
Without an answerfile? It would get stuck asking questions. Was this ever tested?

Perhaps also add 'ignore_errors: true'.

Reply:
yes you right!!
we have an open issue about it in:
https://github.com/oVirt/ovirt-ansible-engine-setup/issues/93

Reply:
I'll move the issue to the collection so we have everything here. (Please let me know if I missed some important issue which should be moved here)

Reply:
https://github.com/oVirt/ovirt-ansible-collection/issues/182